docs/src/inputs/button/devSandbox/examples/exampleOutlineColors.jsx

```diff
@@ -0,0 +1,121 @@
+import {
+    Button,
+    Grid,
+    Icon,
+} from 'react-cm-ui';
+import React from 'react';
+
+function ExampleOutlineColors() {
+    return (
+        <Grid
+            spacing={2}
+        >
+            <Grid.Column
+                sm="auto"
+            >
+                <Button
+                    color="primary"
+                    outline
+                >
+                    <Icon
+                        type="shape-heart"
+                    />
```

Comment:
**_TOTAL NAB_** and 100% discretionary to do anything here.  Should we show `title` props on the `<Icon>`s in these examples?